app-emulation/cri-o/cri-o-1.9.1.ebuild

```diff
@@ -40,7 +45,13 @@ src_compile() {
 	pushd src/${EGO_PN} || die
 	GOPATH="${S}" GOBIN="${S}/bin" \
 		BASE_LDFLAGS=" -s -w -X main.gitCommit=${GIT_COMMIT} -X main.buildInfo=Gentoo" \
-		emake -j1 # -C "${S}/src/${EGO_PN}" dev
+		BUILDTAGS=""
+		if !use btrfs; then BUILDTAGS="${BUILDTAGS} exclude_graphdriver_btrfs"; fi
+		if !use device-mapper; then BUILDTAGS="${BUILDTAGS} exclude_graphdriver_devicemapper"; fi
```

Comment:
after poking around the source tree, I'm not sure this is the correct BUILDTAG, could you point me to some source to assuage my concerns?

Reply:
> after poking around the source tree, I'm not sure this is the correct BUILDTAG, could you point me to some source to assuage my concerns?

It hasn't been backported to 1.9.x, but the motivation is in kubernetes-incubator/cri-o#1268.

Reply:
Well, seeing that it doesn't adversely affect the current build I'm cool with it.